script/Deploy.s.sol

```diff
@@ -1,21 +1,14 @@
 // SPDX-License-Identifier: UNLICENSED
-
-pragma solidity ^0.8.16;
+pragma solidity >=0.8.0;
```

Comment:
Same comment around pragmas as in #13, in general you don't want `>=` unless it's an interface file that supports a range of versions. We don't know whether or not this contract will be compatible with solidity `0.9.0` so we don't want to imply it does with the pragma

Reply:
Ah okay makes sense, I updated all the pragmas in 13.

Reply:
Ok, I'm assuming we'll merge the PRs in order so I'll leave this unresolved until 13 is merged + this rebased on top, to make sure it didn't get reverted in the rebase

Reply:
Sounds good to me

Reply:
Rebased with #13 in main